app/validate_txs.go

```diff
@@ -42,6 +44,7 @@ func filterStdTxs(dec sdk.TxDecoder, ctx sdk.Context, handler sdk.AnteHandler, t
 		// simply want to remove this tx, or we're catching a panic from one
 		// of the anteHanders which is logged.
 		if err != nil {
+			logger.Error("filtering already checked transaction", "tx", tmbytes.FromBytes(tx), "error", err)
```

Comment:
Ideally would prefer not to dump the entire transaction. It wouldn't really help for debugability. I think it would be better either to decode it and print it out or just to print out the message type

Reply:
whoops good catch mean to hash that

Reply:
logging the hash and the msg types for the sdk txs

https://github.com/celestiaorg/celestia-app/pull/2518/commits/12eed6d464767cc1e773b97dc3c5b0639558733b

e486596441113fb01f318ba4d33a7edb85dceec1